contrib/lang/html/packages.el

```diff
@@ -96,3 +96,31 @@ which require an initialization must be listed explicitly in the list.")
 (defun html/init-slim-mode ()
   (use-package slim-mode
     :defer t))
+
+(evil-leader/set-key-for-mode 'web-mode
+  "mgp" 'web-mode-element-parent
+  "mgc" 'web-mode-element-child
+  "mgb" 'web-mode-element-beginning
+  "mgn" 'spacemacs/html-navigation-micro-state
+  "mgs" 'web-mode-element-sibling-next
+  ;; v like 'View'
+  "mvx" 'web-mode-dom-xpath
```

Comment:
Since it prints some info I would put in on `mh`, something like `mhp` or `mhx`.